Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.razor.cs

```diff
@@ -11,6 +11,12 @@ namespace Havit.Blazor.Components.Web.Bootstrap;
 /// <typeparam name="TItem"></typeparam>
 public partial class HxSearchBox<TItem> : IAsyncDisposable
 {
+	protected const string ArrowUpKeyCode = "ArrowUp";
+	protected const string ArrowDownKeyCode = "ArrowDown";
+
+	protected const string EnterKeyCode = "Enter";
+	protected const string NumpadEnterKeyCode = "NumpadEnter";
```

Comment:
Make all the temporary implementation `private` and in single `#region` (if possible), to be able to locate and remove it easily later on.

Reply:
Do we want to do this even though this may end up being the sustained solution?

Reply:
Ok, make it just `private`.